docs/lib/predictions/fragments/ios/identify-text.md

```diff
@@ -35,6 +54,29 @@ func detectText(_ image: URL) {
 }
 ```
 
+To get results that utilize on-device capabilities (Core ML), without combining results from the backend, you can use the following to pass into the `options` argument of the `Amplify.Predictions.identify` function.
+```swift
+let options = PredictionsIdentifyRequest.Options(defaultNetworkPolicy: .offline, pluginOptions: nil)
+```
+
+Note that bounding boxes in `IdentifyTextResult` are returned as ratios.  If you would like to place bounding boxes on an image, use the following method to calculate a frame.  The frame can be used to create view that is overlayed on to the image.
+```swift
+let image = UIImage(...)
+self.imageView = UIImageView(image: self.image)                        
+...
+let data = result as! IdentifyTextResult
+let word = data.words!.first!
+...
+let xBoundingBox = word.boundingBox.origin.x * self.imageView.frame.size.width
+let yBoundingBox = word.boundingBox.origin.y * self.imageView.frame.size.height
+let widthBoundingBox = word.boundingBox.size.width * self.imageView.frame.size.width
+let heightBoundingBox = word.boundingBox.size.height * self.imageView.frame.size.height
+let boundingBox = UIView(frame: CGRect(x: xBoundingBox, y: yBoundingBox, width: widthBoundingBox, height: heightBoundingBox))
+... // stylize bounding box
+self.imageView.addSubview(boundingBox)
+```
+Note that Rekognition places (0,0) at the top left and Core ML places (0,0) at the bottom left, we have flipped the y axis of the CoreML bounding box for you since iOS starts (0,0) from the top left
+
```

Comment:
Not sure if this is still relevant, can someone help me understand if this is even relevant to say anymore?

Reply:
its relevant and this addition is actually more helpful since the sample apps arent available yet